includes/admin/importers/class-wc-product-wp-importer.php

```diff
@@ -15,7 +15,7 @@
  * @package     WooCommerce/Admin/Importers
  * @version     3.1.0
  */
-class WC_Product_Importer extends WP_Importer {
+class WC_Product_WP_Importer extends WP_Importer {
```

Comment:
Do we actually use anything from `WP_Importer` to extend?

> Importer should be exposed in Tools > Import
> Import should also be a submenu item under the Products menu. This will be the primary location.

I'm wondering if our importer should be separated.